oozie-to-airflow/run-sys-test

```diff
@@ -61,12 +61,38 @@ export DATAPROC_USER=${DATAPROC_USER:=${USER:=example_user}}
 
 RUN_SSH_COMMAND="false"
 
+SETUP_CONFIGURATION="false"
+
 SETUP_AUTOCOMPLETE="false"
 
 COMPOSER_WEB_UI_URL=""
 
 VERBOSE="false"
 
+function setup_configuration {
+    read -r -p "Enter Composer name [${COMPOSER_NAME}]: " input
```

Comment:
This should be converted in a separate function (verify_variable) and we should ask for each variable separately if not present/nor read from the file. This will be far more convenient - this way we will automatically ask for every new variable that is added to run-setup-test script in the future.